### PR TITLE
Automated release

### PR DESCRIPTION
**Pull request by gyuho**
Closes: #114

### CODE REVIEW

**Review by dasconnor — APPROVED** [review]
LGTM

**Comment by gyuho** [comment]
Blocked on upcoming duktape deprecation along with some other changes from coreth.

**Comment by gyuho** [comment]
Rebased off https://github.com/ava-labs/subnet-evm/pull/109.

**Comment by gyuho** [comment]
To be merged after https://github.com/ava-labs/subnet-evm/pull/108.

File: tests/e2e/go.mod [hunk 1]
```diff
@@ -3,107 +3,32 @@ module github.com/ava-labs/subnet-evm/tests/e2e
 go 1.17
 
 require (
-	github.com/ava-labs/avalanche-network-runner v1.0.11
-	github.com/onsi/ginkgo/v2 v2.1.3
+	github.com/ava-labs/avalanche-network-runner-sdk v0.0.0
+	github.com/ava-labs/avalanchego v1.7.11-rc.27
```
Comment (patrick-ogrady) [inline tests/e2e/go.mod:7]: Should change to v1.7.11

Comment (ceyonur) [reply tests/e2e/go.mod:7]: created #114

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**